modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/NumericDDMFormFieldTypeUtilTest.java

```diff
@@ -0,0 +1,162 @@
+import org.junit.runner.RunWith;
+
+import org.powermock.modules.junit4.PowerMockRunner;
+
+/**
+ * @author Carolina Barbosa
+ */
+@RunWith(PowerMockRunner.class)
+public class NumericDDMFormFieldTypeUtilTest
+	extends BaseDDMFormFieldTypeSettingsTestCase {
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_setUpLanguageUtil();
+	}
+
+	@Test
+	public void testGetInputMaskParametersDouble() {
+		DDMFormField ddmFormField = new DDMFormField("field", "numeric");
+
+		ddmFormField.setProperty("inputMask", true);
+
+		Map<String, Object> parameters =
+			NumericDDMFormFieldTypeUtil.getParameters(
+				"double", ddmFormField, new DDMFormFieldRenderingContext());
+
+		Assert.assertEquals(parameters.toString(), 6, parameters.size());
+		Assert.assertTrue(parameters.containsKey("append"));
+		Assert.assertTrue(parameters.containsKey("appendType"));
+		Assert.assertTrue(parameters.containsKey("decimalPlaces"));
+		Assert.assertTrue(parameters.containsKey("inputMask"));
+		Assert.assertTrue(parameters.containsKey("numericInputMask"));
+		Assert.assertTrue(parameters.containsKey("symbols"));
+	}
+
+	@Test
+	public void testGetInputMaskParametersInteger() {
+		DDMFormField ddmFormField = new DDMFormField("field", "numeric");
+
+		ddmFormField.setProperty("inputMask", true);
+
+		Map<String, Object> parameters =
+			NumericDDMFormFieldTypeUtil.getParameters(
+				"integer", ddmFormField, new DDMFormFieldRenderingContext());
+
+		Assert.assertEquals(parameters.toString(), 2, parameters.size());
+		Assert.assertTrue(parameters.containsKey("inputMask"));
+		Assert.assertTrue(parameters.containsKey("inputMaskFormat"));
+	}
+
+	@Test
+	public void testGetParametersDouble() {
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext =
+			new DDMFormFieldRenderingContext();
+
+		ddmFormFieldRenderingContext.setLocale(LocaleUtil.US);
+
+		Map<String, Object> parameters =
+			NumericDDMFormFieldTypeUtil.getParameters(
+				"double", new DDMFormField("field", "numeric"),
+				ddmFormFieldRenderingContext);
+
+		Assert.assertEquals(parameters.toString(), 2, parameters.size());
+		Assert.assertEquals(
+			HashMapBuilder.put(
+				"en_US",
+				HashMapBuilder.put(
+					"decimalSymbol", "."
+				).put(
+					"thousandsSeparator", ","
+				).build()
+			).put(
+				"pt_BR",
+				HashMapBuilder.put(
+					"decimalSymbol", ","
+				).put(
+					"thousandsSeparator", "."
+				).build()
+			).build(),
+			parameters.get("localizedSymbols"));
+		Assert.assertEquals(
+			HashMapBuilder.put(
+				"decimalSymbol", "."
+			).put(
+				"thousandsSeparator", ","
+			).build(),
+			parameters.get("symbols"));
+	}
+
+	@Test
+	public void testGetParametersInteger() {
+		Map<String, Object> parameters =
+			NumericDDMFormFieldTypeUtil.getParameters(
+				"integer", new DDMFormField("field", "numeric"),
+				new DDMFormFieldRenderingContext());
+
+		Assert.assertTrue(parameters.isEmpty());
+	}
+
+	private void _setUpLanguageUtil() {
```

Comment:
Would putting this mock (with two available locales and all) in the base `setUpLanguageUtil` method affect other tests?

Reply:
I'm not sure, that's why I didn't put it, but I'll check it out!

Reply:
I'll keep this convo open so you can let me know what was the outcome. 😊

Reply:
Does not affect other tests. I'll move to `BaseDDMFormFieldTypeSettingsTestCase` 😄